packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/webappProvision.template.bicep

```diff
@@ -29,15 +29,20 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
     siteConfig: {
       alwaysOn: true
       appSettings: [
-        {
-          name: 'SCM_DO_BUILD_DURING_DEPLOYMENT'
-          value: 'true' // Execute build steps on your site during deployment
-        }
         {{#if (contains "node" configs)}}
         {
           name: 'WEBSITE_NODE_DEFAULT_VERSION'
           value: '~14' // Set NodeJS version to 14.x for your site
         }
+        {
+          name: 'SCM_SCRIPT_GENERATOR_ARGS'
+          value: '--node' // Register as node server
+        }
+        {{else}}
+        {
```

Comment:
Why not making them consistent? I mean, node (--node) vs dotnet (remote build) are in different behaviors.

Reply:
yes, that's a good idea